packages/dd-trace/src/config.js

```diff
@@ -20,10 +20,10 @@ class Config {
       platform.env('DD_AGENT_HOST'),
       platform.env('DD_TRACE_AGENT_HOSTNAME')
     )
+    const pluginsPlatform = platform.env('DD_TRACE_DISABLED_PLUGINS')
```

Comment:
This option and the `plugins` option are currently unrelated until we implement the object syntax for `plugins`.

`disabledPlugins` would also probably be a better name.

Reply:
Ok understood. I've refactored this once more, back to what was closer to my original implementation, with `disabledPlugins` (and env var `DD_TRACE_DISABLED_PLUGINS` ) as separate from the `plugins` option. `disabledPlugins` now only accepts a csv formatted string of integration names to disable. Going a bit in circles here but I think this feels pretty straightforward for users and allows for future deprecation of the `true/false` `plugins` option without having to worry about this functionality getting lost. Apologies if i'm misunderstanding the intent behind your feedback here, lmk if i'm off base here